src/model/page.urls.ts

```diff
@@ -10,14 +10,18 @@ export const PROMISE_TO_FILE: string = SEPARATOR + "promise-to-file";
 export const COMPANY_NUMBER: string = SEPARATOR + template.COMPANY_NUMBER;
 export const CHECK_COMPANY: string = SEPARATOR + template.CHECK_COMPANY;
 export const COMPANY_AUTH_PROTECTED_ROUTE: string = SEPARATOR + "company/:companyNumber/";
+export const CONFIRMATION_APPEND: string = "confirmation";
 
 /**
  * Company Auth protected routes
  */
 export const STILL_REQUIRED: string = COMPANY_AUTH_PROTECTED_ROUTE + template.STILL_REQUIRED;
-export const CONFIRMATION: string = COMPANY_AUTH_PROTECTED_ROUTE + "confirmation";
```

Comment:
I'd argue to leave this line in over the new version, as the "confirmation" string only appears once (now that we're using string.replace() to create the url), not worth having a constant for "confirmation"